### ISSUE TITLE
[android-5] sox incorrect libmagic linking

### ISSUE THREAD

**Issue by ilyapashuk** [termux/termux-packages#4687]
**Problem description**
A clear and concise description of what the problem is. You may post screenshots in addition to description.
sox binary, and all related binaries, searches for libmagic as libmagic.so.1, when it exists as simply libmagic.so
**Steps to reproduce**
install sox with pkg install sox and try to run it

**Additional information**
when i created symlink from .so to .so.1 file, all start work perfectly.

**Comment by xeffyr** [comment]
@Grimler91 Are you sure that issue affects Android 5 ? The `file`'s debfile seems contain the necessary link:
```
drwx------ builder/builder       0 2019-12-02 10:47 ./data/data/com.termux/files/usr/lib/
-rwx------ builder/builder  133368 2019-12-02 10:47 ./data/data/com.termux/files/usr/lib/libmagic.so.1.0.0
lrwxrwxrwx builder/builder       0 2019-12-02 10:47 ./data/data/com.termux/files/usr/lib/libmagic.so -> libmagic.so.1.0.0
lrwxrwxrwx builder/builder       0 2019-12-02 10:47 ./data/data/com.termux/files/usr/lib/libmagic.so.1 -> libmagic.so.1.0.0
```

**Comment by xeffyr** [comment]
Fixed in `v14.4.2-17`.
